aries-site/src/pages/foundation/color.js

```diff
@@ -1,18 +1,44 @@
 import React from 'react';
+import PropTypes from 'prop-types';
 import { Box, Text } from 'grommet';
-import { Button } from 'aries-core';
+import { Button, AnchorCallToAction } from 'aries-core';
 
 import {
   ColorRow,
   ContentSection,
   PageLayout,
   Subsection,
+  UsageExample,
 } from '../../layouts';
 import { MainDescription, LastUpdated, MainHeading } from '../../components';
-import { greyscaleColors, primaryColors, supportingColors } from '../../data';
+import {
+  greyscaleColors,
+  primaryColors,
+  supportingColors,
+  textColors,
+} from '../../data';
 
 const title = 'Color';
 
+const TextExample = ({ color, hex }) => {
+  return (
+    <Box align="center" margin={{ horizontal: 'small' }}>
+      <Box direction="row" align="center">
+        <Text color={color} weight={700} size="84px">
```

Comment:
The custom sizing and weight doesn't feel right, is it necessary?

Reply:
I think this execution is limited and specific to this particular use case and will not be reused elsewhere. The alternative would be to add an `xxlarge` prop for `Text` on the theme, but this execution would still need to overwrite line-height and font-weight.

Reply:
@ericsoderberghp would love to hear your suggestion here.

Reply:
I've wanted a larger size on Text as well. @L0ZZI and I discussed last week and he was going to work on that. So, I think we need some larger text size options for grommet in general.

Reply:
Sounds good, I filed a separate issue on Grommet, we can come back and refine when more sizes are supported.